src/datasources/asset-calibration/AssetCalibrationDataSource.ts

```diff
@@ -151,6 +166,19 @@ export class AssetCalibrationDataSource extends DataSourceBase<AssetCalibrationQ
     }
   }
 
+  private async loadSystems(): Promise<void> {
+    if (this.state.systems) {
+      return;
+    }
+
+    try {
+      const systems = await this.querySystems('', ['id', 'alias','connected.data.state', 'workspace']);
```

Comment:
I see no reason to keep all the systems information. It can be quite big.

Reply:
Do you need the 'connected.data.state', 'workspace' ?

Reply:
![image](https://github.com/user-attachments/assets/892904d0-c043-4bc1-80f5-62bce392b58b)

these are the required fields on SystemMetadata